src/core/tests/type_prop/slice_scatter.cpp

```diff
@@ -0,0 +1,1055 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/slice_scatter.hpp"
+
+#include <gtest/gtest.h>
+
+#include <tuple>
+
+#include "common_test_utils/test_assertions.hpp"
+#include "common_test_utils/type_prop.hpp"
+#include "openvino/op/constant.hpp"
+#include "sequence_generator.hpp"
+
+using namespace ov;
+using namespace testing;
```

Comment:
Can these using be removed?

Wrap tests inside `ov::test` namespace then using ov should not be required.
From testing take what is required only.